overrides/inspec/property_override.rb

```diff
@@ -23,7 +23,8 @@ class PropertyOverride < Overrides::PropertyOverride
       def self.attributes
         [
           :name_from_self_link, # Set to convert self link to name
-          :exclude_plural
+          :exclude_plural,
```

Comment:
Almost certainly overkill, but fyi:

Ansible has two sets of overrides (one for facts modules, one for regular).

In theory, you could do the same with plural/singular resources + just use the regular exclude field.

Again, almost certainly overkill.